15.2.0-rc5: ncm-lcgbdii broken

This is a showstopper for 15.2.0 release (https://github.com/quattor/release/issues/81). After upgrading to 15.2.0-rc5, `ncm-lcgbdii` fails with the following error despite there was no recent change to the component. A side-effect of `CAF` modifications?

```
2015/03/19-14:59:47 -----------------------------------------------------------
2015/03/19-14:59:47 [ERROR] error filling template /usr/lib/ncm/config/lcgbdii/lcgbdii.template
2015/03/19-14:59:47 [INFO] configure on component lcgbdii executed, 1 errors, 0 warnings
```

--- a/ncm-condorconfig/pom.xml
+++ b/ncm-condorconfig/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-glitestartup/pom.xml
+++ b/ncm-glitestartup/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-lcgbdii/pom.xml
+++ b/ncm-lcgbdii/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-vomrs/pom.xml
+++ b/ncm-vomrs/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-wlconfig/pom.xml
+++ b/ncm-wlconfig/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-wmsclient/pom.xml
+++ b/ncm-wmsclient/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-wmslb/pom.xml
+++ b/ncm-wmslb/pom.xml
@@ -40,5 +40,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

--- a/ncm-yaim/pom.xml
+++ b/ncm-yaim/pom.xml
@@ -47,5 +47,66 @@
     </contributor>
   </contributors>
 
-  
+  <build>
+    <pluginManagement>
+      <plugins>
+
+        <plugin>
+
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+
+          <executions>
+
+            <execution>
+              <id>filter-template-sources</id>
+              <phase>process-sources</phase>
+              <goals>
+                <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                <encoding>UTF-8</encoding>
+                <delimiters>
+                  <delimiter>${*}</delimiter>
+                </delimiters>
+                <outputDirectory>${project.build.directory}/templates</outputDirectory>
+                <resources>
+                  <resource>
+                    <directory>src/main/templates</directory>
+                    <filtering>true</filtering>
+                    <includes>
+                      <include>*</include>
+                    </includes>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+
+          <configuration>
+            <mappings combine.children="append">
+              <mapping>
+                <directory>/usr/lib/ncm/config/${project.artifactId}</directory>
+                <filemode>644</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>target/templates</location>
+                  </source>
+                </sources>
+              </mapping>
+            </mappings>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
 </project>

